graph/src/env.rs

```diff
@@ -36,3 +36,21 @@ pub fn unsafe_env_var<E: std::error::Error + Send + Sync, T: FromStr<Err = E> +
 
     value
 }
+
+/// Panics if:
+/// - The value is not UTF8.
+/// - The value cannot be parsed as T.
+/// - The value differs from the default, and `--unsafe-config` flag is not set.
+pub fn env_var<E: std::error::Error + Send + Sync, T: FromStr<Err = E> + Eq>(
```

Comment:
Finally!

Reply:
Ha yes! That comment shouldn't be mentioning `--unsafe-config`, fixed that.

Reply:
Wow I didn't see this before, this is awesome! 🎉